Add image size to image_detection example index.html

jj-wd opened the pull request:
Added the target image width and height so the model doesn't drift around

blairmacintyre commented:
The size isn't needed on the image in `<img id="hubs-image" src="hubs.png" >` for tracking.  We never display the `img` element in this sample, so the width/height are irrelevant.

The image size for tracking is the last parameter in `this.session.createImageAnchor(hubsImageName, hubsImageData.data, hubsImageData.width, hubsImageData.height, 0.2)` (the 0.2, which is 20cm).

jj-wd commented:
I changed both at the same time and the cause was just changing the 0.2 - as you can see I added a comment before that line as well, which might be useful for others? However, I'll close this. Thanks again!
